### ISSUE TITLE
[PHP] Generating unexpected type hinting

### ISSUE THREAD

**Issue by jfcherng** [20Tauri/DoxyDoxygen#139]
The reproducer is

```php
<?php

class Foo
{
    public function A()
    {
        return null; // this null seems to be problematic
    }

    public function B(?int $bar)
    {
    }
}
```

I tried to add documentation to the method `B` but got

```php
<?php

class Foo
{
    public function A()
    {
        return null; // this null seems to be problematic
    }

    /**
     * { function_description }
     *
     * @param \l $bar The bar
     */
    public function B(?int $bar)
    {
    }
}
```

The type of `$bar` is hinted as unexpected `\l`. The expected type should be something like `int|null` or `?int`.

**Comment by 20Tauri** [comment]
Thank you for reporting,

This case is caused by the FQN option... and it's fixed in the 0.76.1 version (just release)

```php
<?php

class Foo
{
    public function A()
    {
        return null; // this null seems to be problematic
    }

    /**
     * { function_description }
     *
     * @param      integer|null  $bar    The bar
     */
    public function B(?int $bar)
    {
    }
}

?>
```

**Comment by jfcherng** [comment]
Confirmed fixed. Thanks :smile:
